.github/workflows/on-remove.yml

```diff
@@ -172,6 +172,54 @@ jobs:
           CONTENTFUL_MANAGEMENT_ACCESS_TOKEN: ${{ secrets.CONTENTFUL_MANAGEMENT_TOKEN }}
           CONTENTFUL_SPACE_ID: ${{ steps.setup.outputs.gp2-contentful-space-id }}
 
+  analytics-algolia-index:
```

Comment:
it feels like a lot of setup just to delete the index and then do it again for the crn-algolia-index one. I'm guessing even the algolia api key you are passing into it has got access to both this one and regular crn-algolia-index so why not just add one line to the the last step of `crn-algolia-index`?